deckhouse-controller/crds/doc-ru-module.yaml

```
@@ -49,3 +49,9 @@ spec:
                       description: Требование к статусу установки кластера (только для встроенных модулей DKP).
                     modules:
                       description: Список других включенных модулей, которые необходимы для работы модуля.
+              accessibility:
+                type: object
+                description: Настройки доступности.
```

```suggestion
                description: Доступность модуля для использования.
```
Besides availability in Deckhouse editions, are there any plans to add anything else to this section in the future?